### ISSUE TITLE
Update/Remove Speechly Support

### ISSUE THREAD

**Issue by jsandlerus** [JamesBrill/react-speech-recognition#191]
As per the documentation, I'm creating a ticket to remove the Speechly support. As of September 19th, 2023 Speechly was bought by Roblox. https://www.speechly.com/blog/speechly-is-joining-roblox

Documentation:
https://github.com/JamesBrill/react-speech-recognition/blob/master/docs/POLYFILLS.md

**Comment by huynguyen-twentyci** [comment]
Yeah, seems that Speechly is closed the free tier, so no way to add the "free" polyfills now
